content/articles/2018-05-ggplot2-2-3-0.Rmd

```diff
@@ -0,0 +1,126 @@
+ data frames don’t have duplicate column names. Previously ggplot2 just took the 
+ first column with a name, which might have concealed a bug.  
+
+ * We decided that it’s worth breaking a small amount of existing code in the 
+ interests of improving future code. For example, this version of ggplot2 adds 
+ support for [tidy evaluation](https://github.com/tidyverse/ggplot2/blob/master/NEWS.md#tidy-evaluation) 
+ making ggplot2 more compatible with the rest of the tidyverse. This breaks a 
+ small number of packages that use ggplot2 but should affect very little data 
+ analysis code, so we decided that it was worth it.  
+
+### What to do if you run into a breaking change
+
+The maintainers of reverse dependencies that are likely to encounter a breaking 
+change will be contacted. However, it's always possible that we've missed 
+something. If you run into an error when running code that works with ggplot2 
+2.2.1 (the version on CRAN as of this writing), you should look for it in the
+["Breaking changes"](https://github.com/tidyverse/ggplot2/blob/master/NEWS.md#breaking-changes) 
+section of news. If you _can't_ find it in the news, please file an issue - 
+either it's an inadvertent change, or we need to describe it better in the news 
+(or make it easier to find when searching for keywords). As per usual, please 
+do a search of [existing issues](https://github.com/tidyverse/ggplot2/issues) 
+before filing a new one to avoid duplication.
+
+You can also ask for help in [community.rstudio.com](https://community.rstudio.com/).
+
+## New features
+
+### Tidy evaluation
+
+You can now use quasiquotation in `aes()`, `facet_wrap()`, and `facet_grid()`. 
+The formula interface for faceting is not going away, but this is meant to 
+make it easier to create functions around `facet_wrap()` and `facet_grid()`, 
+including label titles for facets with variable names and values.
+
+```{r label_both}
+p <- ggplot(mtcars, aes(wt, mpg)) + geom_point()
+
+p + facet_grid(vars(vs), vars(cyl), labeller = label_both)
+```
+
+### Nonstandard aesthetics
+
+Support for nonstandard aesthetics has been dramatically improved, thanks to
+Claus Wilke, author of the popular [cowplot](https://cran.r-project.org/web/packages/cowplot/vignettes/introduction.html) 
+and [ggridges](https://cran.r-project.org/web/packages/ggridges/vignettes/introduction.html) 
+packages. You can now specify aesthetics independent of the scale name, and 
+specify more than one aesthetic at once, e.g. `aesthetics = c("colour", "fill")`.
+
+```{r density-alpha}
+ggplot(iris, aes(x = Sepal.Length, fill = Species)) +
+  geom_density(alpha = 0.7) +
+  scale_colour_brewer(type = "qual", aesthetics = "fill")
+```
+
+```{r multiple-aes}
+ggplot(iris, aes(x = Species, y = Sepal.Length, color = Species, fill = Species)) +
+  geom_boxplot(alpha = 0.2) +
+  geom_point(position = "jitter") +
+  scale_colour_hue(c = 50, l = 40, aesthetics = c("colour", "fill"))
+```
+
+## And more...
```

Comment:
Can you please add brief sections for `geom_sf()`, `calc()` (soon to be `stat()`), and option for viridis colour scales.